src/Microsoft.Extensions.Caching.Abstractions/DistributedCacheExtensions.cs

```diff
@@ -43,5 +44,93 @@ public static Task SetAsync(this IDistributedCache cache, string key, byte[] val
 
             return cache.SetAsync(key, value, new DistributedCacheEntryOptions());
         }
+
+        /// <summary>
+        /// Sets a string in the specified cache with the specified key.
+        /// </summary>
+        /// <param name="cache">The cache in which to store the data.</param>
+        /// <param name="key">The key to store the data in.</param>
+        /// <param name="value">The data to store in the cache.</param>
+        public static void SetString(this IDistributedCache cache, string key, string value)
+        {
+            cache.SetString(key, value, new DistributedCacheEntryOptions());
+        }
+
+        /// <summary>
+        /// Sets a string in the specified cache with the specified key.
+        /// </summary>
+        /// <param name="cache">The cache in which to store the data.</param>
+        /// <param name="key">The key to store the data in.</param>
+        /// <param name="value">The data to store in the cache.</param>
+        /// <param name="options">The cache options for the entry.</param>
+        public static void SetString(this IDistributedCache cache, string key, string value, DistributedCacheEntryOptions options)
+        {
+            if (value == null)
+            {
+                throw new ArgumentNullException(nameof(value));
+            }
+            cache.Set(key, Encoding.UTF8.GetBytes(value), options);
+        }
+
+        /// <summary>
+        /// Asynchronously sets a string in the specified cache with the specified key.
+        /// </summary>
+        /// <param name="cache">The cache in which to store the data.</param>
+        /// <param name="key">The key to store the data in.</param>
+        /// <param name="value">The data to store in the cache.</param>
+        /// <returns>A task that represents the asynchronous set operation.</returns>
+        public static Task SetStringAsync(this IDistributedCache cache, string key, string value)
+        {
+            return cache.SetStringAsync(key, value, new DistributedCacheEntryOptions());
+        }
+
+        /// <summary>
+        /// Asynchronously sets a string in the specified cache with the specified key.
+        /// </summary>
+        /// <param name="cache">The cache in which to store the data.</param>
+        /// <param name="key">The key to store the data in.</param>
+        /// <param name="value">The data to store in the cache.</param>
+        /// <param name="options">The cache options for the entry.</param>
+        /// <returns>A task that represents the asynchronous set operation.</returns>
+        public static Task SetStringAsync(this IDistributedCache cache, string key, string value, DistributedCacheEntryOptions options)
+        {
+            if (value == null)
```

Comment:
There is no ContainsKey, so storying a null entry is indistinguishable from ignoring/removing the entry. We can store string.Empty as byte[0], but we don't have a meaningful way to serialize and store null.